quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,48 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
```

Comment:
We removed this; you must be using an old version of allennlp.  The tokenizer to use now is the `pretrained-transformer` one.

Reply:
OK, I'm only using the latest *released* version of AllenNLP (0.9.0). To clarify, all the code and course content should be compliant with AllenNLP 1.0.0 correct?

Reply:
Also there was a quite large change in how tokenizers work https://github.com/allenai/allennlp/pull/3361 so we need to change some code in this example repo as well as the course code